src/client.c

```diff
@@ -1754,8 +1769,10 @@ int main(int argc, char *argv[]) {
         { "host", 't', 0, G_OPTION_ARG_STRING_ARRAY, &hostarr,
             "Set host for a recipe with specific id.",
             "<recipe_id>=[<user>@]<host>" },
-        {"conn-retries", 'c', 0, G_OPTION_ARG_INT, &app_data->max_retries,
+        { "conn-retries", 'c', 0, G_OPTION_ARG_INT, &app_data->max_retries,
             "Specify the number of reconnection retries.", NULL},
+        { "port", 'p', 0, G_OPTION_ARG_INT, &app_data->restraint_port,
```

Comment:
An earlier comment requested -p/--port be added to 'restraint' documentation but you said it was defined already.  This code here adds the new option so how could this already be described in the commands.rst doc. for the restraint client  In the commands.rst doc, there are 2 sections: One for restraintd server and a separate section for 'restraint' client.
Also If you are striving to be backward compatible, then perhaps you shouldn't add this option and instead do as previously defined where the port was tagged onto the host.  So the format before Bill's changes was as follows:
--host 1=root@dev-kvm-guest-07.rhts.eng.bos.redhat.com:8082
Do you want to continue to be backward compatible?
In either case, this should be documented for the restraint client.